runtime/oti/j9port_generated.h

```diff
@@ -582,6 +582,7 @@ extern J9_CFUNC int32_t j9port_isCompatible(struct J9PortLibraryVersion *expecte
 #define j9vmem_commit_memory(param1,param2,param3) OMRPORT_FROM_J9PORT(privatePortLibrary)->vmem_commit_memory(OMRPORT_FROM_J9PORT(privatePortLibrary),param1,param2,param3)
 #define j9vmem_decommit_memory(param1,param2,param3) OMRPORT_FROM_J9PORT(privatePortLibrary)->vmem_decommit_memory(OMRPORT_FROM_J9PORT(privatePortLibrary),param1,param2,param3)
 #define j9vmem_free_memory(param1,param2,param3) OMRPORT_FROM_J9PORT(privatePortLibrary)->vmem_free_memory(OMRPORT_FROM_J9PORT(privatePortLibrary),param1,param2,param3)
+#define j9vmem_release_double_mapped_region(param1,param2,param3) OMRPORT_FROM_J9PORT(privatePortLibrary)->vmem_release_double_mapped_region(OMRPORT_FROM_J9PORT(privatePortLibrary),param1,param2,param3)
```

Comment:
Minor rant.... This mapping is probably unnecessary, since at the call site we could create OMR PortLib instance and call OMR API directly .

We already have many instances in OpenJ9 code that we do such things (use directly OMR API of different sorts), but we also have already pre-existing OpenJ9-to-OMR mapping in this file.

I don't know what's preferred way newly introduced APIs. I prefer simplicity (and avoid this mapping). @pshipton @DanHeidinga, opinion?

Reply:
I prefer for new code to use omr directly rather than using a mapping. I believe the mappings were introduced just to avoid changing a lot of code. In the future we may want to remove them.

Reply:
I agree with Peter.  Let's avoid introducing new mappings and use OMR directly when it makes sense

Reply:
Updated `vmem_release_double_mapped_region` calls to use OMR API.